include/nccl_ofi_freelist.h

```diff
@@ -29,6 +29,9 @@ struct nccl_ofi_freelist_elem_t {
 	struct nccl_ofi_freelist_elem_t *next;
 };
 
+_Static_assert(sizeof(struct nccl_ofi_freelist_elem_t) % MEMCHECK_GRANULARITY == 0,
```

Comment:
Don't add an assert unless you have code to make it true.  Don't make the next person do the hard work.